rootfs/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM gcr.io/google_containers/ubuntu-slim:0.5
+FROM ubuntu
```

Comment:
Wouldn't it be safer to use `ubuntu:16.04` explicitly? I know that's what the implicit `:latest` tag points to currently, but it won't always.

https://hub.docker.com/_/ubuntu/